README.md

```diff
@@ -43,6 +43,9 @@ Create a For Loop that outputs the following string for each piece of fruit in t
 
 ```js
 // write code here
+for (i = 0, i <= foods[0].length; i++) {
```

Comment:
Not quite, there are a few issues with this answer: 
1. There is a syntax error on line 46: it should be `for(i = 0; ` not `for(i = 0,` 
2. It should be` i < foods[0].length;` over` i <= foods[0].length;` I would encourage you to test this out in your console
3. There is another syntax error on line 47 with a colon instead of a semicolon